network-api/networkapi/urls.py

```diff
@@ -27,6 +30,9 @@
 
 urlpatterns = list(filter(None, [
     url(r'^admin/', include(admin.site.urls)),
+    url(r'^cms/', include(wagtailadmin_urls)),
+    url(r'^documents/', include(wagtaildocs_urls)),
+    url(r'^wagtail/', include(wagtail_urls)),
```

Comment:
Can/should we feature flag all the things that wagtail does in `urls.py` and `settings.py`? Then we can make sure we only enable it on staging and prod when we're ready to have people log in there and start porting content.

Reply:
I'm fine with that, @alanmoo ?

Reply:
Will the migrations still be able to happen on the production database if we feature flag settings?

Reply:
They'll run, but successfully? They require certain apps to be installed properly, don't they?

Reply:
The migrations should be agnostic to installed apps, since they're working directly with the ORM

Reply:
Ok, let’s do the feature flag but default to “disabled” and let it go to staging that way so we confirm.